analyzer/evmtokens/evm_tokens.go

```diff
@@ -131,25 +131,20 @@ func (m main) processStaleToken(ctx context.Context, batch *storage.QueryBatch,
 			return fmt.Errorf("downloading new token %s: %w", staleToken.Addr, err)
 		}
 		// Use the totalSupply downloaded directly from the chain if it exists,
-		// else default to the dead-reckoned value in analysis.evm_tokens.
-		totalSupplyBytes, err := staleToken.TotalSupply.MarshalText()
-		if err != nil {
-			return fmt.Errorf("error converting totalSupply: %w", err)
-		}
-		totalSupply := string(totalSupplyBytes)
```

Comment:
This is equivalent to `staleToken.String()` from what I can see. I don't know why the convoluted code. Maybe the value was a `big.Int` before (is `common.BigInt` now) and this was needed.